ConcernsCaseWork/ConcernsCaseWork.CypressTests/cypress/integration/casework-regression/CaseActions/nti-under-consideration.spec.ts

```diff
@@ -0,0 +1,140 @@
+import CaseManagementPage from "../../../pages/caseMangementPage";
+import AddToCasePage from "../../../pages/caseActions/addToCasePage";
+import { Logger } from "../../../common/logger";
+import { EditNtiUnderConsiderationPage } from "../../../pages/caseActions/ntiUnderConsideration/editNtiUnderConsiderationPage";
+import { ViewNtiUnderConsiderationPage } from "../../../pages/caseActions/ntiUnderConsideration/viewNtiUnderConsiderationPage";
+import { CloseNtiUnderConsiderationPage } from "../../../pages/caseActions/ntiUnderConsideration/closeNtiUnderConsiderationPage";
+
+describe("Testing the NTI under consideration", () =>
+{
+    const editNtiUnderConsiderationPage = new EditNtiUnderConsiderationPage();
+    const viewNtiUnderConsiderationPage = new ViewNtiUnderConsiderationPage();
+    const closeNtiUnderConsiderationPage = new CloseNtiUnderConsiderationPage();
+
+    beforeEach(() => {
+		cy.login();
+
+        cy.basicCreateCase();
+
+        addNtiUnderConsiderationToCase();
+	});
+
+    describe("When adding or editing a new NTI under consideration", () =>
+    {
+        it("Should create an NTI with the values specified and be able to edit the values", () =>
+        {
+            Logger.Log("Validating notes field");
+            editNtiUnderConsiderationPage
+                .withNotesExceedingLimit()
+                .save()
+                .hasValidationError("Notes provided exceed maximum allowed length (2000 characters).");
+
+            editNtiUnderConsiderationPage
+                .withReasons("Cash flow problems")
+                .withReasons("Safeguarding")
+                .withNotes("These are my notes")
+                .save();
+
+            Logger.Log("Validate the NTI on the view page");
+                cy.get("#open-case-actions td")
+                    .getByTestId("NTI Under Consideration").click();
+
+            viewNtiUnderConsiderationPage
+                .hasReasons("Cash flow problems")
+                .hasReasons("Safeguarding")
+                .hasNotes("These are my notes");
+                
+            viewNtiUnderConsiderationPage.edit();
+
+            editNtiUnderConsiderationPage
+                .hasReasons("Cash flow problems")
+                .hasReasons("Safeguarding")
+                .hasNotes("These are my notes");
+
+            Logger.Log("Validating notes field");
+            editNtiUnderConsiderationPage
+                .withNotesExceedingLimit()
+                .save()
+                .hasValidationError("Notes provided exceed maximum allowed length (2000 characters).");
+
+            editNtiUnderConsiderationPage
+                .clearReasons()
+                .withReasons("Governance concerns")
+                .withReasons("Risk of insolvency")
+                .withNotes("Edited my notes")
+                .save();
+
+            viewNtiUnderConsiderationPage
+                .hasReasons("Governance concerns")
+                .hasReasons("Risk of insolvency")
```

Comment:
Is it possible to check that the other reasons have been removed? I'm assuming this test would still pass if they are still showing. Just a question for the future, not a blocker for the PR

Reply:
Added a length check so it checks only the ones we selected are included